packages/compass-shell/src/components/compass-shell/compass-shell.tsx

```diff
@@ -302,9 +231,12 @@ export class CompassShell extends Component<
   }
 }
 
-export default connect((state: RootState) => ({
-  emitShellPluginOpened: () => {
-    state.runtime.appRegistry?.emit('compass:compass-shell:opened');
```

Comment:
This event was not used by anything in the app, so I completely removed this emit from both versions of the plugin